extensions/Cake.ArtifactDrop.yml

```diff
@@ -0,0 +1,16 @@
+Type: Addin
+Name: Cake.ArtifactDrop
+NuGet: Cake.ArtifactDrop
+Assemblies:
+- /**/Cake.ArtifactDrop.dll
+Repository: 
+ProjectUrl: https://www.nuget.org/packages/Cake.ArtifactDrop/
```

Comment:
```suggestion
ProjectUrl: https://github.com/stefandevo/Cake.ArtifactDrop
```

For some reason AddinDisco didn't pick up the ProjectUrl in the NuGet package which is https://github.com/stefandevo/Cake.ArtifactDrop when opening in the NuGet Gallery

Reply:
Here's the reason why the discoverer did not use the GitHub URL:
![image](https://user-images.githubusercontent.com/112710/112722490-49fbf680-8ee0-11eb-9e7f-910ab34ee895.png)

Reply:
Looks like it's another one for the exclusion list then. The repo doesn't seem to be anywhere on GitHub